src/hotspot/cpu/x86/globalDefinitions_x86.hpp

```diff
@@ -75,4 +75,6 @@ const bool CCallingConventionRequiresIntsAsLongs = false;
 #define COMPRESSED_CLASS_POINTERS_DEPENDS_ON_COMPRESSED_OOPS false
 #endif
 
+#define USE_POINTERS_TO_REGISTER_IMPL_ARRAY
```

Comment:
So, what's the use for this symbol? I see AArch64 code conditionalize definition macros, but this patch does not have such conditionalization. Should it?

Reply:
It means we pick up the correct definitions of `CONSTANT_REGISTER_DECLARATION` et al.

Reply:
Ah, those definitions are in shared code! Nevermind then.